apps/web/src/pages/templates/TemplatesListNoData.tsx

```diff
@@ -146,6 +147,7 @@ export const TemplatesListNoData = ({
           <FontAwesomeIcon icon={faDiagramNext} />
           <span>All templates</span>
         </CardTile>
+        <EchoProjectCardTile />
```

Comment:
I would suggest moving this to the second position

![Screenshot 2024-03-12 at 10 18 52](https://github.com/novuhq/novu/assets/2607232/69252d08-6d6a-4533-b422-f1c8e6eaeae4)